### DISCUSSION TITLE
关于导航的“标签”栏

### DISCUSSION THREAD

**Discussion by fujiwaranatsuho** [MaLuns/hexo-theme-async#102] [Q&A 疑难杂症] [answered]
~~虽然我额外配置和第三方都按照文档写代码但好像一个都没成功总之先问标签~~我设置在“top_bars”下设置title: menu.tags，“menu”下也设置了“tags: 标签”，但在导航栏显示，只有标签是“menu.tags”。请问这是怎么回事呀？

**Comment by MaLuns** [comment, marked as answer]
语言文件里没有这个 menu.tags ，你可以直接把 title 设置成标签 或者 扩展语言文件 增加 menu.tags

**Reply by fujiwaranatsuho** [reply]
谢谢！设置好了。另外想问下，主题有设置显示字数统计吗？我写了相关代码但是页面没有加载，看了几个demo站也没有wordcount。（问题有点多，希望没有特别麻烦你）

**Reply by MaLuns** [reply]
列表是有的，可以文档里搜索下，安装对应插件配置下就可以了

**Reply by fujiwaranatsuho** [reply]
刚才看到了！原来在每篇文章的预览可以看到，点开文章是没有的。实在是太蠢了抱歉抱歉
